credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -138,14 +138,16 @@ func DefaultServerHandshakerOptions() *ServerHandshakerOptions {
 //       and server options (server options struct does not exist now. When
 //       caller can provide endpoints, it should be created.
 
-// altsHandshaker is used to complete a ALTS handshaking between client and
+// altsHandshaker is used to complete an ALTS handshake between client and
 // server. This handshaker talks to the ALTS handshaker service in the metadata
 // server.
 type altsHandshaker struct {
 	// RPC stream used to access the ALTS Handshaker service.
 	stream altsgrpc.HandshakerService_DoHandshakeClient
 	// the connection to the peer.
 	conn net.Conn
+	// the connection to the ALTS handshaker service.
```

Comment:
Nit: a grpc.ClientConn is not a single connection to a service. It is a virtual connection which manages one or more actual connections to backends.

Reply:
Amended the comment for clarity.